pkg/sql/zone_config.go

```diff
@@ -129,6 +126,14 @@ func getZoneConfig(
 		return rootID, zone, placeholderID, placeholder, nil
 	}
 
+	// TODO(arul): For now, secondary tenants get this static default zone config.
+	//  We probably want to seed system.zones with a default zone configuration
+	//  eventually though.
```

Comment:
Instead of this, let's just seed it now. Wouldn't it just be around `p.CreateTenant` in `pkg/sql/tenant.go`?